plugins/vbe/vbe-simple.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * VBE plugin for fwupd,mmc-simple
```

Comment:
I think this should be called `fu-vbe-simple-device.c` -- but also -- why the _simple_? Are there going to be other device types in the future? i.e. could this just be `FuVbeDevice` and `fu-vbe-device.c`?

Reply:
Yes there will be more. Once this is figured out the next one is a verified boot and then A/B/recovery. We may also have ones were we need to talk to some kernel service to initiate an update.

'simple' just means that it is a simple, single firmware image. It covers a large number of devices today, I believe, but these will move to a more secure setup over time.

Reply:
Ack, fu-vbe-simple-device.c then please.

Reply:
done